source/FreeRTOS_IP.c

```diff
@@ -186,6 +186,9 @@ NetworkAddressingParameters_t xNetworkAddressing = { 0, 0, 0, 0, 0 };
 
 /** @brief Default values for the above struct in case DHCP
  * does not lead to a confirmed request. */
+
+/* used for unit testing */
```

Comment:
One might ask - "How is this used for unit-testing?"
Can we maybe make the comment a bit more descriptive? I think we should take a page from FreeRTOS-Kernel about the comments.